services/metadactyl-clj/src/metadactyl/routes/apps/elements.clj

```diff
@@ -1,22 +1,22 @@
 (ns metadactyl.routes.apps.elements
-  (:use [common-swagger-api.schema]
-        [metadactyl.metadata.element-listings :only [list-elements]]
-        [metadactyl.routes.domain.app.element]
-        [metadactyl.routes.domain.tool :only [ToolListing]]
-        [metadactyl.routes.params])
-  (:require [clojure-commons.error-codes :as ce]
-            [metadactyl.util.service :as service]
-            [compojure.route :as route]))
+      (:use [common-swagger-api.schema]
+            [metadactyl.metadata.element-listings :only [list-elements]]
+            [metadactyl.routes.domain.app.element]
+            [metadactyl.routes.domain.tool :only [ToolListing]]
+            [metadactyl.routes.params]
+            [ring.util.http-response :only [ok]])
+      (:require [metadactyl.util.service :as service]
+                [compojure.route :as route]))
 
 (defroutes* app-elements
-  (GET* "/" [:as {uri :uri}]
+  (GET* "/" []
         :query [params SecuredIncludeHiddenParams]
         :summary "List All Available App Elements"
         :description "This endpoint may be used to obtain lists of all available elements that may be
         included in an App."
-        (ce/trap uri #(list-elements "all" params)))
+        (ok (#(list-elements "all" params))))
```

Comment:
I imagine this can just be `(ok (list-elements "all" params))`, not any reason to create an intermediate anonymous function that's just immediately called (similarly below).